src/utils/getWindowDimensions.ts

```diff
@@ -0,0 +1,23 @@
+import { Dimensions } from 'react-native';
+import {getOrientation, getInitialOrientation} from 'react-native-orientation-locker';
```

Comment:
@noumantahir  this import is causing issue. These import do not exist. We have to import Orientation and then destructure it to get its methods